### ISSUE TITLE
Speed up search engine indexing after ingestion?

### ISSUE THREAD

**Issue by nschneid** [acl-org/acl-anthology#2020]
I am wondering if there is a way to trigger an update to the index following the ingestion of a major venue. It is slightly annoying to be at a conference and not be able to use the search box to find papers by title.

For example, https://aclanthology.org/search/?q=double+graph+frame+semantic+parsing does not show a paper being presented today (but the first author has published previously, so I was able to find it at https://aclanthology.org/people/c/ce-zheng/).

**Comment by mbollmann** [comment]
My two cents here are that the only reasonable and reliable way to get this functionality is to maintain our own search, like [reviving the search prototype I made](https://github.com/mbollmann/aclfind), at least for the basics like searching for authors and paper titles.

**Comment by akoehn** [comment]
I agree with @mbollmann -- we had this discussion before (maybe not in a gh issue) and there not that much that can be done on our side short of having our own search engine.
